### PR TITLE
fix: combobox hidden labels and help text describededby attribute and fix token list

### PR DESCRIPTION
**Pull request by SiTaggart**
Description and useful links to discussions / issues / tickets

### Combobox hidden labels and help text accessibility

Improved handling of hidden labels and help text for accessibility reasons. 

### Change Two

Randomly noticed the token list page is a very broken list.

## Checklist

- [ ] Work in a "Draft" branch whilst you are getting feedback to reduce Chromatic costs. Once you are ready for approval, convert the PR to "Ready to review"
- [ ] Ensure all `required` checks have passed
- [ ] Icon changes must have a product design review
- [ ] Website content changes must have a product design review
- [ ] At least two engineers have reviewed any code changes
- [ ] At least one engineering lead has reviewed any core package changes or repository infrastructure
- [ ] Website visual regression testing is run by adding `🕵🏻‍♀️ Run website visual regression` label

### CODE REVIEW

**Review by TheSisb — DISMISSED** [review]

File: packages/paste-core/components/combobox/src/singleselect/Combobox.tsx [hunk 1]
```diff
@@ -166,8 +167,7 @@ const Combobox = React.forwardRef<HTMLInputElement, ComboboxProps>(
               {...getInputProps({disabled, required, ref, ...props})}
               {...(!autocomplete ? {onChange: (event: React.ChangeEvent) => event.preventDefault()} : undefined)}
               autocomplete={autocomplete}
-              aria-describedby={helpTextId}
-              aria-label={hideVisibleLabel ? labelText : null}
+              aria-describedby={helpText != null ? helpTextId : null}
```
Comment (TheSisb) [inline packages/paste-core/components/combobox/src/singleselect/Combobox.tsx:170]: question: why this change?

Comment (SiTaggart) [reply packages/paste-core/components/combobox/src/singleselect/Combobox.tsx:170]: so it was throwing an incomplete check, which I'm guessing it was doing for the hidden label too, where we reference an id that doesn't exist in the DOM if you don't have help text. So it might be an error that would get thrown at some point in the future.

This way it'll never be a problem. It only gets a describebed by attribute if there is a helptext passed in.

**Review by TheSisb — APPROVED** [review]
